src/Avalonia.Controls.TreeDataGrid/Selection/TreeDataGridRowSelectionModel.cs

```diff
@@ -110,6 +113,99 @@ void ITreeDataGridSelectionInteraction.OnKeyDown(TreeDataGrid sender, KeyEventAr
 
         }
 
+        protected void HandleCharKey(string? text, TreeDataGrid treeDataGrid, int selectedRowIndex)
```

Comment:
Should this just be called `OnTextInput`? The name makes me expect that it handles a `Key`, whereas it actually handles text input in the form of a `string`.

Reply:
Well, they don't do different things as such: one is called from the other with additional information. But `HandleTextInput` as you've changed it to is fine ;)